setup.cfg

```diff
@@ -4,6 +4,7 @@ doctests = True
 max_line_length = 160
 max_doc_length = 160
 max-complexity = 25
+ignore=E203
```

Comment:
Where does this lead to problems?
https://www.flake8rules.com/rules/E203.html

Reply:
E203 is a collision between flake8 and black.
https://black.readthedocs.io/en/stable/the_black_code_style.html?highlight=E203#slices

The initial problem occurs [here](https://github.com/dbsystel/gitlab-ci-python-library/blob/fix-linter/gcip/core/cache.py#L181)

Reply:
Sound fine. I would document this functional requirement here.

```suggestion
# https://black.readthedocs.io/en/stable/the_black_code_style.html?highlight=E203#slices
ignore=E203
```

Reply:
I added the comment.